packages/terra-form-input/src/terra-dev-site/doc/form-input/InputField.2.doc.mdx

```diff
@@ -0,0 +1,50 @@
+import { Badge } from 'terra-form-input/package.json?dev-site-package';
+
+import InputFieldExample from '../example/InputField?dev-site-example';
+import RequiredInputFieldExample from '../example/RequiredInputField?dev-site-example';
+import IncompleteInputFieldExample from '../example/IncompleteInputField?dev-site-example';
+import NumberInputFieldExample from '../example/NumberInputField?dev-site-example';
+
+
+import FormInputFieldPropsTable from 'terra-form-input/src/InputField?dev-site-props-table';
+
+<Badge />
+
+# Terra Form Input Field
+
+[terra-form-input](https://github.com/cerner/terra-core/tree/master/packages/terra-form-input) component that is wrapped inside a
+[terra-form-field](https://github.com/cerner/terra-core/tree/master/packages/terra-form-field) component.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-input`
+
+## Usage
+
+```jsx
+import InputField from 'terra-form-input/lib/InputField';
+```
+
+## Component Features
+
+ * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+ * [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+ * [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+ * [LTR/RTL Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#ltr--rtl-support)
+
+<!-- Uncomment supported features.
```

Comment:
lets delete this

Reply:
Deleted here https://github.com/cerner/terra-core/pull/2830/commits/9ac4b85006a3d743adc37cf91dae1de8a0e7d831.